core/components/langrouter/src/Plugins/Events/OnHandleRequest.php

```diff
@@ -83,8 +83,8 @@ public function process()
 
                 // Set locale since $this->modx->_initCulture is called before OnHandleRequest
                 if ($this->modx->context && $this->modx->getOption('setlocale', null, true)) {
-                    $locale = setlocale(LC_ALL, null);
-                    setlocale(LC_ALL, $this->modx->context->getOption('locale', null, $locale));
+                    $locale = setlocale(LC_ALL, 0);
+                    setlocale(LC_ALL, $this->modx->context->getOption('locale', 0, $locale));
```

Comment:
In Zeile 87 aber nicht!

Reply:
Yes, you're right. In line 87 it a mistake from me. The second parameter are the props for `getOption()` 😿